examples/kubernetes-batch-demo/docker/Dockerfile.stepflow-server

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 DataStax Inc.
```

Comment:
Should probably create a "proper" docker build file that we can use for stepflow (eg, multi-stage, cargo-chef, etc.)

Reply:
#352